source/includes/entityref/paybill/_expensesheet.md

```diff
@@ -0,0 +1,146 @@
+# ExpenseSheet
+
+This entity represents the required certifications for a Job Submission.
```

Comment:
Make JobSubmission one word.
There is other inconsistency but that's what we want here.